adal/devApps/AdalUAPTestApp/MainPage.xaml.cs

```diff
@@ -43,7 +43,8 @@ public sealed partial class MainPage : Page
     {
         private const string ClientId = "cd01dc27-9d3c-4812-beda-8229d5d4a8d5";
 
-        private const string ReturnUri = "https://MyDirectorySearcherApp";
+        private readonly Uri ReturnUri = null; // will use the broker (WEB)
+        //private readonly Uri ReturnUri = new Uri("https://MyDirectorySearcherApp"); 
```

Comment:
>//private readonly Uri ReturnUri = new Uri("https://MyDirectorySearcherApp"); [](start = 8, length = 77)

shy is this one commented? Shouldn't it be possible to select one or the other in the App?